packages/webview_flutter/webview_flutter_wkwebview/example/lib/main.dart

```diff
@@ -70,6 +91,7 @@ class _WebViewExampleState extends State<_WebViewExample> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      backgroundColor: Colors.green,
```

Comment:
What about hard coding the color so if it changes in the framework it doesn't break this test in the plugin repo only on the master tests?
```suggestion
      backgroundColor: Color(0xFF4CAF50),
```